src/modules/services_manager_service.rb

```diff
@@ -35,6 +35,7 @@ class ServicesManagerServiceClass < Module
     extend Yast::I18n
 
     SERVICE_SUFFIX = '.service'
+    SOCKET_SUFFIX = '.socket'
```

Comment:
Is this still needed?

Reply:
Nope. I'll remove it in the "final" commit :)